smctemp.cc

```diff
@@ -317,37 +317,37 @@ kern_return_t SmcAccessor::ReadSmcVal(const UInt32Char_t key, SmcVal_t& val) {
   kern_return_t result;
   SmcKeyData_t  inputStructure;
   SmcKeyData_t  outputStructure;
-    
+
   memset(&inputStructure, 0, sizeof(SmcKeyData_t));
   memset(&outputStructure, 0, sizeof(SmcKeyData_t));
   memset(&val, 0, sizeof(SmcVal_t));
 
   inputStructure.key = string_util::strtoul(key, 4, 16);
-  sprintf(val.key, key);
-    
+  snprintf(val.key, 5, key);
```

Comment:
```suggestion
  snprintf(val.key, sizeof(val.key), key);
```
Could you use `sizeof` in second argument?

Reply:
Done!